src/Veldrid/Vk/VkGraphicsDevice.cs

```diff
@@ -506,6 +506,20 @@ private void CreateInstance(bool debug, VulkanDeviceOptions options)
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
+                // The new MoltenVK now requires the following code
+                if (availableInstanceExtensions.Contains(CommonStrings.VK_KHR_portability_enumeration))
+                {
+                    instanceExtensions.Add(new FixedUtf8String("VK_KHR_portability_enumeration"));
+
+                    //
+                    // Vulkan.RawConstants.VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR
+                    // does not exist
+                    //
+                    // #define VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR = 0x00000001
```

Comment:
nit: I'd prefer if this was an actual constant in the code and referenced in the assignment on 520.

Reply:
I think this constant should be added to the Vk project, not the Veldrid project.
However, I feel that this would have a large impact.